inc/search.php

```diff
@@ -19,15 +19,15 @@
 
 add_action( 'rest_api_init', function() {
 
-	register_rest_route( 'hm-handbook/v1', '/search', array(
+	register_rest_route( 'hm-handbook/v1', '/search', [
```

Comment:
Didn't know about short array syntax. Is it safe to use? Looks like it was introduced in PHP 5.4, so maybe avoid for public work?